src/abstract-classes/browser-controller.js

```diff
@@ -11,15 +11,19 @@ class BrowserController extends EventEmitter {
     constructor(options) {
         super();
 
-        const { browser, proxyUrl, browserPlugin } = options;
+        const { browser, proxyUrl, anonymizedProxy, browserPlugin, ...rest } = options;
```

Comment:
The problem here is that we would have to then return the Context with the BrowserController in the `launch`.  I feel like the BrowserController itself should have its context properties as it has the `browser` property. In order to avoid the `Object.assign()`. We could add a context object to the BrowserController.